src/style/string-sheet.js

```diff
@@ -32,6 +32,7 @@ styfn.appendFromString = function( string ){
     }
   }
 
+  // eslint-disable-next-line
```

Comment:
I generally don't like ignoring an entire line for all rules.  I think it's better to selectively disable the rule of interest in case there's another actual problem in the line in future.